### PR TITLE
Add license

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by isuruf** [comment]
@jakirkham, do you have an idea why appveyor fails?

**Comment by jakirkham** [comment]
Seems it can't find the license file. 😕

```python
IOError: [Errno 2] No such file or directory: u'C:\\Miniconda\\conda-bld\\work\\COPYING.LESSER'
```

xref: https://ci.appveyor.com/project/conda-forge/mpfr-feedstock/build/1.0.21/job/86ob6t65yyo9h52w#L1967

**Comment by jakirkham** [comment]
Restarting AppVeyor to see if the issue was spurious.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -45,6 +45,7 @@ test:
 about:
   home: http://www.mpfr.org/
   license: LGPL 3
+  license_file: COPYING.LESSER
```
Comment (jakirkham) [inline recipe/meta.yaml]: Maybe try `license_file: {{ SRC_DIR }}/COPYING.LESSER`.

**Comment by isuruf** [comment]
Tests pass now. It was because of a missing include, but the error shown was unrelated.

**Pull request merged**